stdlib/cmake/modules/SwiftSource.cmake

```diff
@@ -1001,7 +1001,9 @@ function(_compile_swift_files
     if(SWIFTFILE_STATIC)
       set(command_copy_interface_file)
       if(interface_file)
-        set(command_copy_interface_file COMMAND "${CMAKE_COMMAND}" "-E" "copy" ${interface_file} ${interface_file_static})
+        set(command_copy_interface_file
```

Comment:
@DougGregor and I spoke about this - we need to build things in two forms, not just copy the swift interfaces.